src/internal/packager/sbom/viewer/viewer.js

```diff
@@ -1,36 +1,36 @@
-const sbomSelector = document.getElementById('sbom-selector')
-const distroInfo = document.getElementById('distro-info')
-const modal = document.getElementById('modal')
-const modalFader = document.getElementById('modal-fader')
-const modalTitle = document.getElementById('modal-title')
-const modalContent = document.getElementById('modal-content')
-const artifactsTable = document.createElement('table')
+const sbomSelector = document.getElementById("sbom-selector")
+const distroInfo = document.getElementById("distro-info")
+const modal = document.getElementById("modal")
+const modalFader = document.getElementById("modal-fader")
+const modalTitle = document.getElementById("modal-title")
+const modalContent = document.getElementById("modal-content")
+const artifactsTable = document.createElement("table")
 
-document.body.appendChild(artifactsTable)
+document.body.appendChild(artifactsTable);
 
 function initSelector() {
     window.location.href
 
     const url = /sbom-viewer-(.*).html*$/gmi.exec(window.location.href)[1];
 
     ZARF_SBOM_LIST.sort().forEach(item => {
-        let selected = (url === item) ? 'selected' : '';
+        let selected = (url === item) ? "selected" : "";
         sbomSelector.add(new Option(item, item, selected, selected));
     });
 }
 
 function initData() {
-    const payload = ZARF_SBOM_DATA
+    const payload = ZARF_SBOM_DATA;
```

Comment:
Glad the `;` are being added, should they also be added in lines 1-7 if we're doing it?